biomage/experiment/utils.py

```diff
@@ -158,30 +158,43 @@ def create_gem2s_hash(experiment, project, samples):
     if experiment['meta']['M']['organism'].get('S'):
         organism = experiment['meta']['M']['organism']['S']
 
-    sample_ids = [sample_id for sample_id in samples['M']]
+    # Filter 'ids' key which is present in older samples object
+    unsorted_sample_ids = [
+        sample_id
+        for sample_id
+        in samples['M']
+        if sample_id != 'ids'
```

Comment:
The line splitting here is kind of weird and hard to read, could you leave it a little more like it was originally?

Reply:
The linter complains because the line is too long. Does this change make it make easier to read?